src/form-field/interfaces.ts

```diff
@@ -60,6 +60,11 @@ export interface FormFieldProps extends BaseComponentProps {
    */
   constraintText?: React.ReactNode;
 
+  /**
+   * Provides a text alternative for the error icon in the error message.
+   */
+  errorIconLabel?: string;
```

Comment:
I would recommend to name it `errorIconAriaLabel`, to keep consistent with other ariaLabel prop we have such as `externalIconAriaLabel`

Reply:
Are we sure we will always use aria-label? I can see a situation where this, for example, becomes visually hidden text in the future. I'm happy to change it if preferred, but I'm wary of baking in implementation assumptions.

Reply:
It is good point. However it is still good to differentiate with visible label by naming. In addition, using aria-label or hidden text make no difference to screenreader users. Even we change it in the future(which I don't yet see why), it is still not bad to keep naming it with aria-label to make the usage of the prop easier to understand. 
Usually when it comes to api change we want more team input to reach consensus. Let's ask @timogasda @avinashbot wdyt?

Reply:
A common approach to this in component libraries is to include an "a11y" prefix or suffix like `a11yErrorIconLabel` or `errorIconLabelA11y`.

Reply:
Generally, I would go for consistency. Unfortunately, we currently don't have consistency for this type of property name when we look at other components. For example, we have `dismissAriaLabel` in Alert, but `dismissLabel` in Flashbar. Personally, I prefer `errorIconLabel` because it does abstract from the implementation, which I think makes sense here.